src/language/__tests__/parser-test.ts

```diff
@@ -120,6 +120,13 @@ describe('Parser', () => {
     });
   });
 
+  it('does not allow "true" as enum value', () => {
+    expectSyntaxError('enum Test { VALID, true }').to.deep.equal({
+      message: 'Syntax Error: Name "true" is not a valid enum value.',
```

Comment:
Maybe we can be more specific and just say it's a restricted name that you can't use as a name of enum value.

Reply:
> Name "true" is reserved and cannot be used as enum value.

I'd keep referring to it as "enum value" instead of "name of an enum value" because it's actually both the name and the value of the enum value (🤯) and calling it "value" is shorter.

Reply:
ah, I an stay ambiguous with that respect and (hopefully) clearer at the same time with

> Name "true" is reserved and cannot be used *for* an enum value.